sections/05-adc-data-publishing.qmd

```diff
@@ -167,7 +207,7 @@ Depending on your internet connection, the number of files you have, and the cum
 Before you can upload your data to the Data Team's server, make sure to email us at `support@arcticdata.io` to retrieve the login password. Once you have that, you can proceed through the following steps.
 
 ::: callout-note
-Please note, if you have multiple terabytes of data, it may be best to arrange a shipment of an external hard drive.
+Please note, if you have data nearing the petabyte scale, it may be best to arrange a shipment of an external hard drive.
 :::
 
 When you start Cyberduck, you should see a screen similar to the image below.
```

Comment:
I think we should remove the explicit Cyberduck instructions - save them for the training manual though

Reply:
Yes, I have the cyberduck stuff copied to a totally different qmd file. I kept it in because I wasn't sure if there was any language at the beginning I wanted to retain, but I will remove all the cyberduck instructions